codec-http2/src/test/java/io/netty/handler/codec/http2/Http2FrameCodecTest.java

```diff
@@ -227,7 +230,7 @@ public void sendGoAway() throws Exception {
         Http2GoAwayFrame goAwayFrame = new DefaultHttp2GoAwayFrame(Http2Error.NO_ERROR.code(), debugData.slice());
         goAwayFrame.setExtraStreamIds(2);
 
-        inboundHandler.writeOutbound(releaseLater(goAwayFrame));
+        inboundHandler.writeOutbound(goAwayFrame);
```

Comment:
@Scottmitch nope I think wrapping it with releaseLater(...) was a bug as this should be handled already.